### PR TITLE
Adding GitHub Actions

### PR DESCRIPTION
**Pull request by rudeboybert**
As described in [Github actions with R](https://ropenscilabs.github.io/actions_sandbox/packageci.html#actions-for-continuous-integration-checks)

### CODE REVIEW

**Comment by rudeboybert** [comment]
Can't make sense of the GitHub Actions error. Do you know off-hand what the fix is?

**Comment by ismayc** [comment]
Seems like the {vctrs} package didn't install? Maybe try a forced install of that via GitHub Actions?

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@master
+      - name: Install dependencies
+        run: |
+          install.packages(c("remotes", "rcmdcheck"))
```
Comment (rudeboybert) [inline .github/workflows/R-CMD-check.yaml]:
```suggestion
          install.packages(c("remotes", "rcmdcheck", "vctrs"))
```

**Comment by rudeboybert** [comment]
Oh dang, I didn't realize you had to scroll down to see the actual error. I'm still so used to the extra-long travis-ci output dumps. Added `vctrs` in 7d3e4994be215f8e8d39fb635b0905ec2b9acdd7

**Comment by rudeboybert** [comment]
Fixed!

**Review by ismayc — APPROVED** [review]

**Pull request merged**